src/useContext.ts

```diff
@@ -0,0 +1,17 @@
+/**
+ * External dependencies
+ */
+import React from "react";
+
+/**
+ * Internal dependencies
+ */
+import { Context } from "./Context";
+
+export const useContext = () => {
```

Comment:
I wish I could come up with a better name that didn't require me to prefix everything with `Media*`.